parsl/executors/high_throughput/executor.py

```diff
@@ -60,6 +60,13 @@ class HighThroughputExecutor(ParslExecutor, RepresentationMixin):
                      +----update_fut-----+
 
 
+    Each of the workers in each process_worker_pool has access to its local rank through
+    an environmental variable, ``PARSL_WORKER_RANK``. The local rank is unique for each process
```

Comment:
I think RANK is a bit overloaded and can be a bit confusing.
What do you think about `PARSL_WORKER_ID`, `PARSL_WORKER_POOL_ID` (as-is) and `PARSL_WORKER_POOL_SIZE` ?

Reply:
Or even, `PARSL_WORKER_RANK` and `PARSL_WORKER_COUNT` might be clearer.

Reply:
This is a good point. I did struggle with figuring out what to name the environmental variables.

I would also vote to stick with `PARSL_WORKER_RANK` and `PARSL_WORKER_COUNT`.

Rank is indeed overload, but I was considering that a benefit for people recognizing the purpose of this environmental variable. The issue is that the rank is local to a single executor and some users might expect it to be global for the global pool of executors. However, I like it better than `ID` because I know to expect `RANK` to start at 0 and increase sequentially.